docs/data/base/components/checkbox/checkbox.md

```diff
@@ -9,108 +9,78 @@ waiAria: https://www.w3.org/WAI/ARIA/apg/patterns/checkbox/
 
 # Checkbox
 
-<p class="description">Checkboxes give users binary choices when presented with multiple options in a series.</p>
+<p class="description">Checkbox gives users a binary choice between multiple options in a series.</p>
 
 {{"component": "modules/components/ComponentLinkHeader.js", "design": false}}
 
 {{"component": "modules/components/ComponentPageTabs.js"}}
 
-## Introduction
-
-The Checkbox component provides users with a checkbox for toggling a checked state.
-
 {{"demo": "UnstyledCheckboxIntroduction", "defaultCodeOpen": false, "bg": "gradient"}}
 
-## Component
+## Installation
 
-```jsx
-import * as Checkbox from '@base_ui/react/Checkbox';
-```
+Base UI components are all available as a single package.
 
-### Anatomy
+<codeblock storageKey="package-manager">
 
-The `Checkbox` component is composed of a root component and an indicator child component:
-
-```tsx
-<Checkbox.Root>
-  <Checkbox.Indicator />
-</Checkbox.Root>
+```bash npm
+npm install @base_ui/react
 ```
 
-The indicator can contain children, such as an icon:
-
-```tsx
-<Checkbox.Root>
-  <Checkbox.Indicator>
-    <MyCheckIcon />
-  </Checkbox.Indicator>
-</Checkbox.Root>
+```bash yarn
+yarn add @base_ui/react
 ```
 
-The indicator conditionally unmounts its children when the checkbox is unchecked. For CSS animations, you can use the `keepMounted` prop to transition `visibility` and `opacity` for example:
-
-```tsx
-<Checkbox.Root>
-  <Checkbox.Indicator keepMounted>
-    <MyCheckIcon />
-  </Checkbox.Indicator>
-</Checkbox.Root>
+```bash pnpm
+pnpm add @base_ui/react
 ```
 
-### Custom structure
+</codeblock>
 
-Use the `render` prop to override the rendered checkbox or indicator element with your own components:
+Once you have the package installed, import the component.
 
-```jsx
-<Checkbox.Root render={(props) => <MyCheckbox {...props} />}>
-  <Checkbox.Indicator render={(props) => <MyCheckboxIndicator {...props} />} />
-</Checkbox.Root>
+```ts
+import * as Checkbox from '@base_ui/react/Checkbox';
 ```
 
-To ensure behavior works as expected:
```

Comment:
IMO this part is quite important. We shouldn't remove it (unless we link to a page that describes the render prop in more detail).